packages/vue/src/components/search/DataSearch.jsx

```diff
@@ -202,6 +205,17 @@ const DataSearch = {
 			this.queryOptions = { ...queryOptions, ...this.getBasicQueryOptions() };
 			this.setQueryOptions(this.$props.componentId, this.queryOptions);
 		},
+		getComponent(downShiftProps = {}) {
+			const data = {
+				...downShiftProps,
+				loading: this.isLoading,
+				currentValue: this.$data.currentValue,
+				suggestions: this.suggestions,
+				parsedSuggestions: this.suggestionsList,
+				aggregationData: this.aggregationData,
+			};
```

Comment:
@ShahAnuj2610 Please return the same properties we used to return in `react`.
```
                         error,
			loading: isLoading,
			downshiftProps,
			data: this.parsedSuggestions,
			aggregationData,
			rawData: this.props.suggestions || [],
			value: currentValue,
			triggerClickAnalytics: this.triggerClickAnalytics,
```

Reply:
I deliberately did that, since `renderAllSuggestions` was behaving the same way and not to break existing user behaviour

Reply:
@ShahAnuj2610 Please make those changes, we can have breaking changes.

Reply:
ok